examples/ex5p.cpp

```diff
@@ -107,7 +107,7 @@ int main(int argc, char *argv[])
    ParMesh *pmesh = new ParMesh(MPI_COMM_WORLD, *mesh);
    delete mesh;
    {
-      int par_ref_levels = 2;
+      int par_ref_levels = 1;
```

Comment:
Revert changes before merging.

Reply:
Changed :-)

Reply:
I think it is ready - Let me know if anything should be changed in order for the code to compliant with MFEM settings.